package.json

```diff
@@ -17,6 +17,7 @@
     "test:native": "./scripts/cpputest.sh",
     "test:junit": "./scripts/cpputest.sh --ci",
     "pretest:asan": "npm run build:asan",
+    "pretest:js": "npm run build",
```

Comment:
why? is the same as `pretest:js-unit`

Reply:
probably is due to the rebase. This PR was quite old so an old change made it to the PR. I will remove it